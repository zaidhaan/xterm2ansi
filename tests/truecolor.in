[48;2;0;0;0m [48;2;1;0;0m [48;2;2;0;0m [48;2;3;0;0m [48;2;4;0;0m [48;2;5;0;0m [48;2;6;0;0m [48;2;7;0;0m [48;2;8;0;0m [48;2;9;0;0m [48;2;10;0;0m [48;2;11;0;0m [48;2;12;0;0m [48;2;13;0;0m [48;2;14;0;0m [48;2;15;0;0m [48;2;16;0;0m [48;2;17;0;0m [48;2;18;0;0m [48;2;19;0;0m [48;2;20;0;0m [48;2;21;0;0m [48;2;22;0;0m [48;2;23;0;0m [48;2;24;0;0m [48;2;25;0;0m [48;2;26;0;0m [48;2;27;0;0m [48;2;28;0;0m [48;2;29;0;0m [48;2;30;0;0m [48;2;31;0;0m [48;2;32;0;0m [48;2;33;0;0m [48;2;34;0;0m [48;2;35;0;0m [48;2;36;0;0m [48;2;37;0;0m [48;2;38;0;0m [48;2;39;0;0m [48;2;40;0;0m [48;2;41;0;0m [48;2;42;0;0m [48;2;43;0;0m [48;2;44;0;0m [48;2;45;0;0m [48;2;46;0;0m [48;2;47;0;0m [48;2;48;0;0m [48;2;49;0;0m [48;2;50;0;0m [48;2;51;0;0m [48;2;52;0;0m [48;2;53;0;0m [48;2;54;0;0m [48;2;55;0;0m [48;2;56;0;0m [48;2;57;0;0m [48;2;58;0;0m [48;2;59;0;0m [48;2;60;0;0m [48;2;61;0;0m [48;2;62;0;0m [48;2;63;0;0m [48;2;64;0;0m [48;2;65;0;0m [48;2;66;0;0m [48;2;67;0;0m [48;2;68;0;0m [48;2;69;0;0m [48;2;70;0;0m [48;2;71;0;0m [48;2;72;0;0m [48;2;73;0;0m [48;2;74;0;0m [48;2;75;0;0m [48;2;76;0;0m [48;2;77;0;0m [48;2;78;0;0m [48;2;79;0;0m [48;2;80;0;0m [48;2;81;0;0m [48;2;82;0;0m [48;2;83;0;0m [48;2;84;0;0m [48;2;85;0;0m [48;2;86;0;0m [48;2;87;0;0m [48;2;88;0;0m [48;2;89;0;0m [48;2;90;0;0m [48;2;91;0;0m [48;2;92;0;0m [48;2;93;0;0m [48;2;94;0;0m [48;2;95;0;0m [48;2;96;0;0m [48;2;97;0;0m [48;2;98;0;0m [48;2;99;0;0m [48;2;100;0;0m [48;2;101;0;0m [48;2;102;0;0m [48;2;103;0;0m [48;2;104;0;0m [48;2;105;0;0m [48;2;106;0;0m [48;2;107;0;0m [48;2;108;0;0m [48;2;109;0;0m [48;2;110;0;0m [48;2;111;0;0m [48;2;112;0;0m [48;2;113;0;0m [48;2;114;0;0m [48;2;115;0;0m [48;2;116;0;0m [48;2;117;0;0m [48;2;118;0;0m [48;2;119;0;0m [48;2;120;0;0m [48;2;121;0;0m [48;2;122;0;0m [48;2;123;0;0m [48;2;124;0;0m [48;2;125;0;0m [48;2;126;0;0m [48;2;127;0;0m [0m
[0m
[48;2;0;0;0m [48;2;0;1;0m [48;2;0;2;0m [48;2;0;3;0m [48;2;0;4;0m [48;2;0;5;0m [48;2;0;6;0m [48;2;0;7;0m [48;2;0;8;0m [48;2;0;9;0m [48;2;0;10;0m [48;2;0;11;0m [48;2;0;12;0m [48;2;0;13;0m [48;2;0;14;0m [48;2;0;15;0m [48;2;0;16;0m [48;2;0;17;0m [48;2;0;18;0m [48;2;0;19;0m [48;2;0;20;0m [48;2;0;21;0m [48;2;0;22;0m [48;2;0;23;0m [48;2;0;24;0m [48;2;0;25;0m [48;2;0;26;0m [48;2;0;27;0m [48;2;0;28;0m [48;2;0;29;0m [48;2;0;30;0m [48;2;0;31;0m [48;2;0;32;0m [48;2;0;33;0m [48;2;0;34;0m [48;2;0;35;0m [48;2;0;36;0m [48;2;0;37;0m [48;2;0;38;0m [48;2;0;39;0m [48;2;0;40;0m [48;2;0;41;0m [48;2;0;42;0m [48;2;0;43;0m [48;2;0;44;0m [48;2;0;45;0m [48;2;0;46;0m [48;2;0;47;0m [48;2;0;48;0m [48;2;0;49;0m [48;2;0;50;0m [48;2;0;51;0m [48;2;0;52;0m [48;2;0;53;0m [48;2;0;54;0m [48;2;0;55;0m [48;2;0;56;0m [48;2;0;57;0m [48;2;0;58;0m [48;2;0;59;0m [48;2;0;60;0m [48;2;0;61;0m [48;2;0;62;0m [48;2;0;63;0m [48;2;0;64;0m [48;2;0;65;0m [48;2;0;66;0m [48;2;0;67;0m [48;2;0;68;0m [48;2;0;69;0m [48;2;0;70;0m [48;2;0;71;0m [48;2;0;72;0m [48;2;0;73;0m [48;2;0;74;0m [48;2;0;75;0m [48;2;0;76;0m [48;2;0;77;0m [48;2;0;78;0m [48;2;0;79;0m [48;2;0;80;0m [48;2;0;81;0m [48;2;0;82;0m [48;2;0;83;0m [48;2;0;84;0m [48;2;0;85;0m [48;2;0;86;0m [48;2;0;87;0m [48;2;0;88;0m [48;2;0;89;0m [48;2;0;90;0m [48;2;0;91;0m [48;2;0;92;0m [48;2;0;93;0m [48;2;0;94;0m [48;2;0;95;0m [48;2;0;96;0m [48;2;0;97;0m [48;2;0;98;0m [48;2;0;99;0m [48;2;0;100;0m [48;2;0;101;0m [48;2;0;102;0m [48;2;0;103;0m [48;2;0;104;0m [48;2;0;105;0m [48;2;0;106;0m [48;2;0;107;0m [48;2;0;108;0m [48;2;0;109;0m [48;2;0;110;0m [48;2;0;111;0m [48;2;0;112;0m [48;2;0;113;0m [48;2;0;114;0m [48;2;0;115;0m [48;2;0;116;0m [48;2;0;117;0m [48;2;0;118;0m [48;2;0;119;0m [48;2;0;120;0m [48;2;0;121;0m [48;2;0;122;0m [48;2;0;123;0m [48;2;0;124;0m [48;2;0;125;0m [48;2;0;126;0m [48;2;0;127;0m [0m
[0m
[48;2;0;0;0m [48;2;0;0;1m [48;2;0;0;2m [48;2;0;0;3m [48;2;0;0;4m [48;2;0;0;5m [48;2;0;0;6m [48;2;0;0;7m [48;2;0;0;8m [48;2;0;0;9m [48;2;0;0;10m [48;2;0;0;11m [48;2;0;0;12m [48;2;0;0;13m [48;2;0;0;14m [48;2;0;0;15m [48;2;0;0;16m [48;2;0;0;17m [48;2;0;0;18m [48;2;0;0;19m [48;2;0;0;20m [48;2;0;0;21m [48;2;0;0;22m [48;2;0;0;23m [48;2;0;0;24m [48;2;0;0;25m [48;2;0;0;26m [48;2;0;0;27m [48;2;0;0;28m [48;2;0;0;29m [48;2;0;0;30m [48;2;0;0;31m [48;2;0;0;32m [48;2;0;0;33m [48;2;0;0;34m [48;2;0;0;35m [48;2;0;0;36m [48;2;0;0;37m [48;2;0;0;38m [48;2;0;0;39m [48;2;0;0;40m [48;2;0;0;41m [48;2;0;0;42m [48;2;0;0;43m [48;2;0;0;44m [48;2;0;0;45m [48;2;0;0;46m [48;2;0;0;47m [48;2;0;0;48m [48;2;0;0;49m [48;2;0;0;50m [48;2;0;0;51m [48;2;0;0;52m [48;2;0;0;53m [48;2;0;0;54m [48;2;0;0;55m [48;2;0;0;56m [48;2;0;0;57m [48;2;0;0;58m [48;2;0;0;59m [48;2;0;0;60m [48;2;0;0;61m [48;2;0;0;62m [48;2;0;0;63m [48;2;0;0;64m [48;2;0;0;65m [48;2;0;0;66m [48;2;0;0;67m [48;2;0;0;68m [48;2;0;0;69m [48;2;0;0;70m [48;2;0;0;71m [48;2;0;0;72m [48;2;0;0;73m [48;2;0;0;74m [48;2;0;0;75m [48;2;0;0;76m [48;2;0;0;77m [48;2;0;0;78m [48;2;0;0;79m [48;2;0;0;80m [48;2;0;0;81m [48;2;0;0;82m [48;2;0;0;83m [48;2;0;0;84m [48;2;0;0;85m [48;2;0;0;86m [48;2;0;0;87m [48;2;0;0;88m [48;2;0;0;89m [48;2;0;0;90m [48;2;0;0;91m [48;2;0;0;92m [48;2;0;0;93m [48;2;0;0;94m [48;2;0;0;95m [48;2;0;0;96m [48;2;0;0;97m [48;2;0;0;98m [48;2;0;0;99m [48;2;0;0;100m [48;2;0;0;101m [48;2;0;0;102m [48;2;0;0;103m [48;2;0;0;104m [48;2;0;0;105m [48;2;0;0;106m [48;2;0;0;107m [48;2;0;0;108m [48;2;0;0;109m [48;2;0;0;110m [48;2;0;0;111m [48;2;0;0;112m [48;2;0;0;113m [48;2;0;0;114m [48;2;0;0;115m [48;2;0;0;116m [48;2;0;0;117m [48;2;0;0;118m [48;2;0;0;119m [48;2;0;0;120m [48;2;0;0;121m [48;2;0;0;122m [48;2;0;0;123m [48;2;0;0;124m [48;2;0;0;125m [48;2;0;0;126m [48;2;0;0;127m [0m
[0m
[48;2;255;0;0m [48;2;255;5;0m [48;2;255;11;0m [48;2;255;17;0m [48;2;255;23;0m [48;2;255;29;0m [48;2;255;35;0m [48;2;255;41;0m [48;2;255;47;0m [48;2;255;53;0m [48;2;255;59;0m [48;2;255;65;0m [48;2;255;71;0m [48;2;255;77;0m [48;2;255;83;0m [48;2;255;88;0m [48;2;255;94;0m [48;2;255;100;0m [48;2;255;106;0m [48;2;255;112;0m [48;2;255;118;0m [48;2;255;124;0m [48;2;255;130;0m [48;2;255;136;0m [48;2;255;142;0m [48;2;255;148;0m [48;2;255;154;0m [48;2;255;160;0m [48;2;255;166;0m [48;2;255;171;0m [48;2;255;177;0m [48;2;255;183;0m [48;2;255;189;0m [48;2;255;195;0m [48;2;255;201;0m [48;2;255;207;0m [48;2;255;213;0m [48;2;255;219;0m [48;2;255;225;0m [48;2;255;231;0m [48;2;255;237;0m [48;2;255;243;0m [48;2;255;249;0m [48;2;255;255;0m [48;2;250;255;0m [48;2;244;255;0m [48;2;238;255;0m [48;2;232;255;0m [48;2;226;255;0m [48;2;220;255;0m [48;2;214;255;0m [48;2;208;255;0m [48;2;202;255;0m [48;2;196;255;0m [48;2;190;255;0m [48;2;184;255;0m [48;2;178;255;0m [48;2;172;255;0m [48;2;167;255;0m [48;2;161;255;0m [48;2;155;255;0m [48;2;149;255;0m [48;2;143;255;0m [48;2;137;255;0m [48;2;131;255;0m [48;2;125;255;0m [48;2;119;255;0m [48;2;113;255;0m [48;2;107;255;0m [48;2;101;255;0m [48;2;95;255;0m [48;2;89;255;0m [48;2;84;255;0m [48;2;78;255;0m [48;2;72;255;0m [48;2;66;255;0m [48;2;60;255;0m [48;2;54;255;0m [48;2;48;255;0m [48;2;42;255;0m [48;2;36;255;0m [48;2;30;255;0m [48;2;24;255;0m [48;2;18;255;0m [48;2;12;255;0m [48;2;6;255;0m [48;2;0;255;0m [48;2;0;255;5m [48;2;0;255;11m [48;2;0;255;17m [48;2;0;255;23m [48;2;0;255;29m [48;2;0;255;35m [48;2;0;255;41m [48;2;0;255;47m [48;2;0;255;53m [48;2;0;255;59m [48;2;0;255;65m [48;2;0;255;71m [48;2;0;255;77m [48;2;0;255;83m [48;2;0;255;88m [48;2;0;255;94m [48;2;0;255;100m [48;2;0;255;106m [48;2;0;255;112m [48;2;0;255;118m [48;2;0;255;124m [48;2;0;255;130m [48;2;0;255;136m [48;2;0;255;142m [48;2;0;255;148m [48;2;0;255;154m [48;2;0;255;160m [48;2;0;255;166m [48;2;0;255;171m [48;2;0;255;177m [48;2;0;255;183m [48;2;0;255;189m [48;2;0;255;195m [48;2;0;255;201m [48;2;0;255;207m [48;2;0;255;213m [48;2;0;255;219m [48;2;0;255;225m [48;2;0;255;231m [48;2;0;255;237m [48;2;0;255;243m [0m
[0m
