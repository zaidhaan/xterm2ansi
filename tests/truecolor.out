[40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [41m [0m
[0m
[40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [42m [0m
[0m
[40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [40m [44m [44m [44m [44m [44m [44m [44m [44m [0m
[0m
[101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [101m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [103m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [43m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [102m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [46m [106m [106m [106m [106m [106m [106m [106m [0m
[0m
